piety-promoted-v1/modernized/16-chapter-15.adoc

```diff
@@ -565,9 +564,12 @@ and a prisoner near five years; aged about seventy-one years.
 
 He died the 12th, and was buried the 14th of the Eighth month, 1706.
 
-Here follows a Testimony of an ancient friend and acquaintance of R. Huberstie`'s.`'
+[.embedded-content-document.testimony]
+--
 
-'`Since I have had the opportunity of reading the above-written lines,
+Here follows a Testimony of an ancient friend and acquaintance of R. Huberstie`'s.
```

Comment:
I would probably mark this line as a LETTER-HEADING, since its kinda describing the paragraph that follows. 👍🏻